packages/sns/src/sns.wrapper.ts

```diff
@@ -156,12 +165,19 @@ export class SnsWrapper {
         owner: this.canisterIds.governanceCanisterId,
         subaccount,
       };
-      let balance = await this.ledger.balance({ ...account, certified: false });
-      if (balance === BigInt(0)) {
-        // Recheck the balance with an update call
-        balance = await this.ledger.balance({ ...account, certified: true });
-        // If the balance is still 0, we can use this subaccount
-        if (balance === BigInt(0)) {
+      const neuronId: SnsNeuronId = { id: subaccount };
+      let neuron = await this.governance.queryNeuron({
+        neuronId,
+        certified: false,
+      });
+      if (neuron === undefined) {
+        // Recheck with update call whether the neuron does not exist
+        neuron = await this.governance.queryNeuron({
```

Comment:
Using the balance to check whether there was a neuron in that account was not correct.

Neurons can have 0 stake. For example, after disbursing.